DataStucture_& _ModernOperator/script.js

```diff
@@ -0,0 +1,94 @@
+// DataStucture_& _ModernOperator practice
+
+const hotelDetails = {
+  name: "taj",
+  Startermenu: ["garlic bread", "manchau suup", "tomato suupp"],
+  menu: ["pizza", "bread", "puff", "panipuri"],
+
+  startTime: {
+    mon: {
+      open: 11,
+      close: 5,
+    },
+    fri: {
+      open: 12,
+      close: 6,
+    },
+    sat: {
+      open: 1,
+      close: 24,
+    },
+  },
+  oerder({ index, address, time }) {
+    console.log(
+      ` ${this.name}  order ${this.Startermenu[index]} or ${this.menu[index]} dilivered to the ${address} at ${time}`
+    );
+  },
+};
+
+hotelDetails.oerder({
+  index: 2,
+  address: "simform solution",
+  time: 20,
+});
+
+const { name, Startermenu } = hotelDetails;
+
+console.log(name, Startermenu);
+
+let [bread, supp] = Startermenu;
+
+console.log(bread, supp);
+
+// switch the variable
+
+[bread, supp] = [supp, bread];
+
+console.log(bread, supp);
+
+// spread Operator practice
+const array = [1, 2, 3, 4, 5, 6, 7];
+const [x, y, z] = [...array];
```

Comment:
why are you doing like this ? what if you do like `const [x, y, z] = array;`

Reply:
No, I was just checking if i pass ...array in [ ] how it will behave. It was just for practice purpose.